ferraris-enumerator.ino

```diff
@@ -49,6 +49,10 @@ std::uint8_t switches[] = {D2, D3, D4, D5, D6, D7, D9, D10};
 
 void setup()
   {
+  gDisplay.power(true);
+  gDisplay.setBrightness(6);
+  gDisplay.writeInteger(9999);
```

Comment:
We could also try to write "Init":
```
|  _  | |_
| | | | |_
```

Reply:
I like that! Although maybe with a lowercase i?

```
|  _    |_
| | | | |_
```

Reply:
Lowercase seems to look better, but better test on the real device :wink: